tests/testGroupScoreAndSync.js

```diff
@@ -0,0 +1,355 @@
+var ERR = require('async-stacktrace');
+
+const config = require('../lib/config');
+const _ = require('lodash');
+const path = require('path');
+var assert = require('chai').assert;
+var request = require('request');
+var cheerio = require('cheerio');
+
+var sqldb = require('@prairielearn/prairielib').sqldb;
+var sqlLoader = require('@prairielearn/prairielib').sqlLoader;
+var sql = sqlLoader.loadSqlEquiv(__filename);
+var page, form, elemList; 
+const helperServer = require('./helperServer');
+const courseDir = path.join(__dirname, '..', 'testCourse');
+
+const locals = {};
+locals.helperClient = require('./helperClient');
+locals.siteUrl = 'http://localhost:' + config.serverPort;
+locals.baseUrl = locals.siteUrl + '/pl';
+locals.courseInstanceUrl = locals.baseUrl + '/course_instance/1';
+locals.courseInstanceBaseUrl = locals.baseUrl + '/course_instance/1';
+
+const storedConfig = {};
+const question = [{qid: 'addNumbers', type: 'Freeform', maxPoints:5}];
+const questions = _.keyBy(question, 'qid');
+
+describe('assessment instance group synchorization test', function () {
+    this.timeout(10000);
+
+    before('set authenticated user', function(callback) {
+        storedConfig.authUid = config.authUid;
```

Comment:
Is this `storedConfig` used anywhere?  Can it be removed?

Reply:
For current test I created, there is no use on change back to dev which means that can be removed. Thanks!